.github/workflows/release.yml

```
@@ -114,3 +114,23 @@ jobs:
           asset_path: ./ittapi_build_${{ github.ref_name }}.zip
           asset_name: ittapi_build_${{ github.ref_name }}.zip
           asset_content_type: application/zip
+  
+  # upload the built wheels to pypi
+  upload_to_pypi:
+    name: Upload the wheel files to pypi
+    runs-on: ubuntu-latest
+    needs: [python_build, create_release]
+    steps:
+      - name: Download artifacts
+        uses: actions/download-artifact@6b208ae046db98c579e8a3aa621ab581ff575935 # v4.1.1
+        with:
+          pattern: build-artifacts-*
+          merge-multiple: true
+      - name: List wheel files
+        run: find -name *.whl
+      - name: Publish the package to pypi
+        uses: pypa/gh-action-pypi-publish@release/v1
+        with:
+          password: ${{ secrets.PYPI }}
+          repository-url: https://upload.pypi.org/legacy/
+          packages-dir: python_dist
```

```suggestion
          packages-dir: python_dist
          skip-existing: true
```
Since our Python bindings have their own version that is not linked to the main ittapi release version we should add the `skip-existing: true` option here to handle the situation when a .whl package with current version is already exists on PyPI. This option tells `twine` tool to skip uploading files that already exist on the repository, thus preventing duplication errors.
https://github.com/marketplace/actions/pypi-publish#tolerating-release-package-file-duplicates

---

I have made the change. Please have a look.